KosherCocoa/Library/Core/Solar/KCNOAACalculator.h

```diff
@@ -0,0 +1,164 @@
+//
+//  KCNOAACalculator.h
+//  KosherCocoa
+//
+//  Created by Elyahu on 1/23/23.
+//
+
+@import Foundation;
+#import "KCGeoLocation.h"
+#import "KCConstants.h"
+#import "KCAstronomicalCalculator.h"
+
+/** A class that uses the US National Oceanicnand Atmospheric Administration Algorithm to calculate sunrise and sunset. */
```

Comment:
nit: Would be great to have a link to an algorithm on Wikipedia or an NOAA website, but totally optional.